Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx

```diff
@@ -0,0 +1,268 @@
+import { NeutralColors } from '@uifabric/fluent-theme';
+import { RouteComponentProps } from '@reach/router';
+import { IPivotItemProps, Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { csharpFeedKey, nodeFeedKey } from '@botframework-composer/types';
+
+import { DialogCreationCopy, EmptyBotTemplateId, feedDictionary, QnABotTemplateId } from '../../../constants';
+import httpClient from '../../../utils/httpUtil';
+
+import { TemplateDetailView } from './TemplateDetailView';
+
+// -------------------- Styles -------------------- //
+
+const detailListContainer = css`
+  width: 50%;
+  height: 400px;
+  overflow: hidden;
+  float: left;
+  flex-grow: 1;
+`;
+
+const templateDetailContainer = css`
+  width: 50%;
+  height: 400px;
+  overflow: auto;
+  flex-grow: 1;
+  float: left;
+`;
+
+const pickerContainer = css`
+  position: relative;
+  height: 400px;
+  border: 1px solid #f3f2f1;
+`;
+
+const rowDetails = (disabled) => {
+  return {
+    root: {
+      color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+      selectors: {
+        '&:hover': {
+          background: disabled ? NeutralColors.white : NeutralColors.gray30,
+          color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+        },
+        '&.ms-DetailsRow.is-selected': {
+          background: disabled ? NeutralColors.white : NeutralColors.gray30,
+          color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+        },
+      },
+    },
+  };
+};
+
+const rowTitle = (disabled) => {
+  return {
+    cellTitle: {
+      color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+      selectors: {
+        ':hover': {
+          background: disabled ? NeutralColors.white : NeutralColors.gray30,
+          color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+        },
+      },
+    },
+  };
+};
+
+const tableCell = css`
+  outline: none;
+  :focus {
+    outline: rgb(102, 102, 102) solid 1px;
+  }
+`;
+
+const content = css`
+  outline: none;
+`;
+
+const optionKeys = {
+  createFromScratch: 'createFromScratch',
+  createFromQnA: 'createFromQnA',
+  createFromTemplate: 'createFromTemplate',
+};
+
+// -------------------- CreateOptions -------------------- //
+type CreateOptionsProps = {
+  templates: BotTemplate[];
+  onDismiss: () => void;
+  onNext: (data: string) => void;
+  fetchTemplates: (feedUrls?: string[]) => Promise<void>;
+} & RouteComponentProps<{}>;
+
+export function CreateOptionsV2(props: CreateOptionsProps) {
+  const [option] = useState(optionKeys.createFromTemplate);
+  const [disabled] = useState(false);
+  const { templates, onDismiss, onNext } = props;
+  const [templateReadMe, setTemplateReadMe] = useState('');
+  const [currentTemplate, setCurrentTemplate] = useState('');
+  const [emptyBotKey, setEmptyBotKey] = useState('');
+  const [selectedFeed, setSelectedFeed] = useState<{ props: IPivotItemProps }>({ props: { itemKey: csharpFeedKey } });
+
+  const selection = useMemo(() => {
```

Comment:
`selection` of what? It's opaque. 

How about `selectedTemplate` or something more descriptive.

Reply:
Handled